spread.yaml

```diff
@@ -877,6 +879,8 @@ suites:
             . "$TESTSLIB/nested.sh"
             nested_destroy_vm
             nested_cleanup_env
+
+            tests.backup restore
```

Comment:
I would prefer to first review a tool and only then use it across the test suite.

Reply:
ok, I'll remove the tool usage